provision/testdata/flavor_cko_aci.inp.yaml

```diff
@@ -0,0 +1,83 @@
+aci_config:
+  system_id: bm2acicni
+  apic_hosts:                  
+    - 10.30.120.100
+  apic_login:
+    username: admin
+    password: test
+  aep: sauto_aep                   
+  vrf:                               
+    name: sauto_l3out-1_vrf      
+    tenant: common
+  l3out:                             
+    name: sauto_l3out-1
+    external_networks:
+        - sauto_l3out-1_epg
+
+connectivity_checker:
+ error_pods_reporting_enable: true
+ reachability_test_enable:
+  periodic_sync:
+   enable_updates: true
+   interval: 120
+  external_url:
+   url: 'google.com'
+  proxy: ""
+ reachability_tests:
+  pod_to_pod:
+    update_status: true
```

Comment:
We dont need update_status field under every test in the input file. We can just capture the "pod_to_pod", pod_to_clusterIP and so on